### PR TITLE
Enable reading sources with configurable delimiter.

### PR DESCRIPTION
**Pull request by gijskant**

### CODE REVIEW

File: tests/sources2csr/test_sources2csr.py [hunk 1]
```diff
@@ -58,3 +65,27 @@ def test_missing_identifier(tmp_path):
...
+    target_path = tmp_path.as_posix()
+    reader = SourcesReader(
+        input_dir='./test_data/input_data/CLINICAL',
+        output_dir=target_path,
+        config_dir='./test_data/input_data/config/invalid_sources_config/wrong_file_format')
+    with pytest.raises(DataException) as excinfo:
+        reader.read_subject_data()
+    # The columns in the biosource.csv file are not correctly parsed, resulting
+    # in a missing identifier
+    assert 'Empty identifier' in str(excinfo.value)
```
Comment (ewelinagr) [inline tests/sources2csr/test_sources2csr.py]: I wonder if there is a way to return some better tips as an error message, when there is an issue with a file_format configuration (typo in a file name etc.). It's easy to forget about checking this.

Comment (gijskant) [reply tests/sources2csr/test_sources2csr.py]: Changed it to missing column with a hint at the sources config.

**Comment by gijskant** [comment]
Note added in the README.

File: README.rst [hunk 2]
```diff
@@ -125,9 +126,12 @@ a JSON file that contains two attributes:
 
 * ``codebooks``: a map from input file name to codebook file name, e.g., ``{"individual.tsv": "codebook.txt"}``.
 
-See `<test_data/input_data/config/sources_config.json>`_ for an example.
+* ``file_format``: a map from input file name to file format options, e.g., ``{"individual.tsv": {"delimiter": ","}}``.
```
Comment (ewelinagr) [inline README.rst]: Maybe also useful to mention that if not specified, it will use `\t` as default?

**Pull request merged**